docker/test/Dockerfile

```diff
@@ -0,0 +1,83 @@
+# https://hub.docker.com/r/browserless/chrome/dockerfile/
+FROM ubuntu:18.04
+
+# Dependencies + NodeJS
+RUN apt-get -qq update && \
+  echo "ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true" | debconf-set-selections && \
+  apt-get -y -qq install software-properties-common &&\
+  apt-add-repository "deb http://archive.canonical.com/ubuntu $(lsb_release -sc) partner" && \
+  apt-add-repository ppa:malteworld/ppa && apt-get -qq update && apt-get -y -qq install \
+  libappindicator3-1 \
+  pdftk \
+  unzip \
+  locales \
+  gconf-service \
+  libasound2 \
+  libatk1.0-0 \
+  libc6 \
+  libcairo2 \
+  libcups2 \
+  libdbus-1-3 \
+  libexpat1 \
+  libfontconfig1 \
+  libgcc1 \
+  libgconf-2-4 \
+  libgdk-pixbuf2.0-0 \
+  libglib2.0-0 \
+  libgtk-3-0 \
+  libnspr4 \
+  libpango-1.0-0 \
+  libpangocairo-1.0-0 \
+  libstdc++6 \
+  libx11-6 \
+  libx11-xcb1 \
+  libxcb1 \
+  libxcomposite1 \
+  libxcursor1 \
+  libxdamage1 \
+  libxext6 \
+  libxfixes3 \
+  libxi6 \
+  libxrandr2 \
+  libxrender1 \
+  libxss1 \
+  libxtst6 \
+  ca-certificates \
+  libappindicator1 \
+  libnss3 \
+  lsb-release \
+  xdg-utils \
+  wget \
+  xvfb \
+  iproute2 \
+  curl &&\
+  # Install Node
+  curl --silent --location https://deb.nodesource.com/setup_10.x | bash - &&\
+  apt-get -y -qq install nodejs &&\
+  apt-get -y -qq install build-essential
+
+# It's a good idea to use dumb-init to help prevent zombie chrome processes.
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+
+ADD ./ /opt/app
+WORKDIR /opt/app
+RUN npm install
+
+# Build Args
+ARG USE_BROWSER='firefox'
```

Comment:
@makarandp0 Why are we hard-coding this value?

Reply:
it's default value for args and not strictly necessary. It will be overwritten by the parameter passed at build time - and would then be baked into the docker image.

The idea is each docker image is built for specific combination of BROWSER+version 
the BROWSER and version to use is determined by the parameter. If you forget to pass the browser+version, it would default to these values. 

Docker-compose passes these arguments at build time by looking at your env variables. 
https://github.com/twilio/twilio-video.js/blob/JSDK-2469_dockerized_tests/docker-compose.yml#L13

but somebody were to use this DockerFile directly (manually) then its good to have some defaults (instead of building image w/o browser)

Reply:
Default in Karma config is chrome. Should we default this to chrome as well?

Reply:
Its best to setup env variables/.env files. I intentionally kept defaults to atypical  ff/unstable so that the developer does not forget them. ( tests are more likely to fail in firefox/unstable :)